Dockerfile

```diff
@@ -18,4 +18,7 @@ RUN dnf install --best --refresh -y \
     && dnf clean all \
     && rm -rf /var/cache/yum \
 
+COPY ./network-snapshot.sh /
+ENTRYPOINT ["/network-snapshot.sh"]
```

Comment:
I prefer adding this to some bin path and use as a tool wherever needed because this docker image is not used just for network debugging anymore.

Reply:
I will remove the trailing slash however I'm not very familiar with dockerfiles. Could you be more specific about the best way to include this script as a bin path? I will re-do the commits and squash (clean up). Thanks!

Reply:
you can try something like this:
```
COPY --chmod=777 network-snapshot.sh /usr/local/bin/network-snapshot.sh
```

and remove ENTRYPOINT, anyone can call the script directly `network-snapshot.sh`